pkgs/by-name/sw/sway-audio-idle-inhibit/package.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, libpulseaudio
+, wayland
+, wayland-protocols
+}:
+stdenv.mkDerivation {
+  pname = "sway-audio-idle-inhibit";
+  version = "0.1.1-unstable-2023-08-09";
```

Comment:
```suggestion
  version = "unstable-2023-08-09";
```

Reply:
Why this change because as far as I've read when there is a version tag that should also be part of the version? It's actually mentioned under the conventions section here: [https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md)

Just asking so I can learn from it (: